src/DTS/Render/Renderer.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+
+namespace HomeCEU\DTS\Render;
+
+use mikehaertl\wkhtmlto\Pdf;
+use Ramsey\Uuid\Uuid;
+
+class Renderer {
+  protected const PDF_OPTIONS = [
+      'no-outline',
+      'exclude-from-outline',
+      'margin-bottom' => 0,
+      'margin-right'  => 10,
+      'margin-left'   => 5,
+      'commandOptions' => ['enableXvfb' => true]
+  ];
+
+  public static function create(): self {
+    return new self();
```

Comment:
open closed principle ... to make this class extendable you should return a new static() instead of self()